pkg/storage/bucket/s3/bucket_client.go

```diff
@@ -37,7 +37,6 @@ func newS3Config(cfg Config) (s3.Config, error) {
 	if err != nil {
 		return s3.Config{}, err
 	}
-
```

Comment:
Maybe we can revert this file to `main`'s version to avoid adding the unneeded change here?